synapse/config/key.py

```diff
@@ -105,6 +132,13 @@ def read_config(self, config, config_dir_path, **kwargs):
                     % (type(key_servers).__name__,)
                 )
 
+            for server in key_servers:
```

Comment:
again: I think this could do with being after the `_perspectives_to_key_servers` logic so that `perspectives` are covered in the warning too.